src/core/atomic.d

```diff
@@ -1341,16 +1336,15 @@ version( unittest )
     }
 
 
-    void testType(T)( T val = T.init + 1 ) pure nothrow
+    void testType(T)( T val = T.init + 1 ) pure nothrow @nogc @safe
     {
         testCAS!(T)( val );
         testLoadStore!(MemoryOrder.seq, T)( val );
         testLoadStore!(MemoryOrder.raw, T)( val );
     }
 
-
-    //@@@BUG@@@ http://d.puremagic.com/issues/show_bug.cgi?id=8081
-    /+pure nothrow+/ unittest
+@safe
+    pure nothrow unittest
```

Comment:
The added `@safe` here is causing problems in some builds. But I can't figure out why. The error is on line 1386, can't take the address of a local in a `@safe` function.